wix/Main.wxs.template

```diff
@@ -1,238 +1,232 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
-  <?include $(sys.CURRENTDIR)\Workdir\$(var.OutputBaseFilename)-$(var.ProductSku).Variables.wxi ?>
-
-  <Product Id="$(var.ProductId)" Name="$(var.ProductNameWithVersion)" Language="$(var.ProductLanguage)" Version="$(var.ProductVersion)" Manufacturer="$(var.ProductManufacturer)" UpgradeCode="$(var.ProductUpgradeCode)">
-    <Package Description="$(var.PackageDescription)" Manufacturer="$(var.ProductManufacturer)" InstallerVersion="200" Compressed="yes" />
-    <MajorUpgrade DowngradeErrorMessage="!(loc.DowngradeDenied)"/>
-    <Media Id="1" Cabinet="Data1.cab" EmbedCab="yes" CompressionLevel="high" />
-    <Property Id="ARPHELPLINK" Value="!(loc.ProductHelpLink)" />
-    <Property Id="ARPURLINFOABOUT" Value="!(loc.ProductUrlInfoAbout)" />
-    <Property Id="ARPURLUPDATEINFO" Value="!(loc.ProductUrlUpdateInfo)" />
-    <Property Id="ALLUSERS" Value="2" /> <!-- Allow per User or per Machine setup -->
```

Comment:
@gdams @douph1 I had to change the name of the property `ALLUSERS` to avoid a duplicate symbol error. I also changed all references to this property, but I now see that this property is an  actual reserved property name that MSI uses. I am still looking for a solution that allows me to set the `ALLUSER` property (the SetProperty tag didn't seem to work), but I wanted to ask if there were any other ideas

Reply:
@jmjaffe37 have you asked upstream? Might be worth discussing this with the wix maintainers

Reply:
@gdams, Is there a slack channel I could ask in? Or should I open an issue in the wix github repo: https://github.com/wixtoolset/wix

Reply:
I asked here this morning, will update when we find a solution:
https://github.com/orgs/wixtoolset/discussions/8151

Reply:
Ended up finding out that it is equivalent to set the Package `Scope` to `perUserOrMachine`, fix has been implemented